### ISSUE TITLE
Randomized edges for fillstitch?

### ISSUE THREAD

**Issue by Augusa** [inkstitch/inkstitch#2208]
The new release for satin with randomized edges works really very well. But what happened with the fillstitch? In the first realise it was possible to get randomized edges for fillstitch but now it is no more possible. I miss this feature.

**Comment by lexelby** [comment]
Ah, thank you!  Are you referring to fill stitch, or E-stitch satin?

**Comment by Augusa** [comment]
On the beginning it is fill stitch which had randimized edges in earlier releases but I don't find it now
For the E-stitch it appears using randomized edges for columns. It is not possible for E-stitch.

**Comment by kaalleen** [comment]
I don't think we ever had a branch with random fill stitch. But I think @claudinepeyrat06 wanted to to work on it...

**Comment by claudinepeyrat06** [comment]
@kaalleen in my very first trial i did start working with fill stitch, but it needs improvement.....

**Comment by rlevine** [comment]
Another vote for random fill here. I started poking at it, but my time evaporated. 

Just for grins, here are some other missing fill features that I've seen in patterns in the wild and are available in other systems. The image below is from Bonnie Nielsen's book "Punch," and I'm using her terms to label the variations. She gives a nice overview of common features without highlighting specific software, although her bias is towards what she uses in her work. (And apologies if we can already do these, and I'm too new to the system to have found them.)

- Our current fill is the "straight" variant here, with each line capped by a perpendicular turn at the edges of a shape. The end lines reinforce the outer shape of the fill.
- I've been able to get part of the way to the "acute" variant, with parallel stitch lines, by skipping small stitches, but that only skips stitches at one end or the other of a row, giving some of the chisel end effect, but needing manual work to adjust the rest of the line ends. (Eliminating the perpendicular transition stitches is valuable to stop the fill from telegraphing the outline of the shape.)
- "Smooth" is the equivalent of our satin stitch, applied to a wider fill. 
- "Jagged" is like satin with random edges. The interesting addition to that would be being able to have one edge smooth and one random.

And two other changes:

- Being able to have random length stitches, as mentioned in this string, to do fills without any tatami patterns.  The tatami patterns can distract from using sparse fills for shading or gradients, showing background fabric or underlying stitches.
- Using curved guides, in the same way we use rungs in satin columns, to guide the stitch lines in a fill, allowing variable and curved stitch lines to follow shapes for modeling and emphasis.   

![fill_variations](https://user-images.githubusercontent.com/2294569/232129782-c5980e92-5b71-4d2c-9e8d-ac8917108bec.jpg)

**Comment by george-steel** [comment]
Another thing for my to-do list.
